### ISSUE TITLE
AI core shutters on metastation prevent syndieborgs from getting in

### ISSUE THREAD

**Issue by Shapsy** [tgstation/-tg-station#13618]
We've established that bolting and depowering your doors roundstart as AI is a bad thing to do since it's metagaming and completely shuts down your core to syndicate cyborgs (in the same vein as atmos techs disabling the plasma lines in atmos), but then there's this dumb shutter in the AI core on meta that does the same thing and pretty much serves no other purpose

**Comment by KorPhaeron** [comment]
Lets add shutters to other maps instead and ops can spend 1tc on C4 if they want to kill an AI

**Comment by swankcookie** [comment]
yeah, let's make AI even more difficult to kill

**Comment by Cebasiz** [comment]
as a constant AI player i'm not really against direct buffs, but why all this sudden backlash against ops? are they winning too much?

i mean, the reason ops kill the AI in the first place is because it can be a legendary pain in the ass. without killing it or subverting it, victory is almost impossible due to emags not working on bolted doors, firelocks and shutters constantly closing. and that's just asimov, if you get robocopped or something it gets much worse.

if you make it so that an operative physically present at the satellite is required to kill the AI you're going to weaken the op team considerably. the only reason the borg is effective is because it can usually kill the AI before it screams and the station has time to ready themselves for the blitzkrieg. if that stops being a thing, op winrates are probably going to tank.

tl;dr why all the changes?

**Comment by swankcookie** [comment]
@Cebasiz While I would tend to agree with you, I believe it has to do with their 45% Win rate, which is only surpassed by Ling.

**Comment by Lumipharon** [comment]
@KorPhaeron Lucky that ops are the only ones that would ever have a need to break into the AI core right?

**Comment by KorPhaeron** [comment]
@Lumipharon 

Only newfriends go in from the front anyway

**Comment by Lumipharon** [comment]
Or people with RCD's. Or without space suits.

Also with shutters, either you let people close them FNR, which is the same as letting AI's bolt/depowering doors (which also locks out syndie borgs), with the side effect of making the core way more robust in other round types (since it can't be hacked open), 
or you don't, in which case it does literally nothing to stop syndie borgs (other then making possibly 1 seconds faster to secure your core if you happen to see the synie borg coming, which is rare).

**Comment by CPTANT** [comment]
I seriously detest the mindset that a syndiborg should be a 100% secure way to kill an AI. 

Ops have shitloads of ways to kill an AI, not being able to do it with the completely braindead method of walking a syndi borg in is hardly a nerf to them.

**Comment by Shapsy** [comment]
It's still a 50TC investment to kill the AI, and if the AI is quick enough to shout out about the syndieborg killing it, the rest of the crew gets a massive warning

**Comment by Lumipharon** [comment]
It's not a problem with how easy it is to kill the AI as a borg though.
nke ops only get syndie borgs to kill the AI because how OVERPOWERED AS FUCK the AI is against nuke ops.

Ops are absolutely shooting themselves in the foot if they don't take out the AI (assuming it's semi non-retarded), and taking it out with anything other then a borg WILL give the AI times to alert the crew.

**Comment by AlexanderUlanH** [comment]
You can't hack open a bolt/depower door.

**Comment by Lumipharon** [comment]
You can (it's just extremely inpractical), it requires more tools and time however.

**Comment by Shadowlight213** [comment]
@lumipharon 
How? Only way to get through a bolt depowered door is ti blow it up or go around. You can't open it  with tools.

**Comment by TheNightingale** [comment]
I'm pretty sure you can emag a shutter to force-open it, by the way.

**Comment by Lumipharon** [comment]
cut the power and backup power wires then mend restores power when you cut power with a multitool.
I can't say with 100 certainty that it works if the AI does it (also cut the ai wire), but it should.

If the AI cuts enviro power, you're shit out of luck though.

**Comment by Metacide** [comment]
Can't you crowbar the shutter anyway if it's depowered? The shutter should be part of the foyer area allowing you to turn off the APC outside the core to depower and open it. Then you're onto the standard bolts issue that is treated normally.

I guess it's a layer of meta that people aren't so familiar with. If I've accidentally put the shutter inside the core area, that's more of an issue perhaps.

**Comment by Metacide** [comment]
Yeah, double-checked this an the APC outside can be disabled to depower the shutter, so this isn't an issue and can be closed.
